book/src/solana-scripts.md

```diff
@@ -0,0 +1,108 @@
+# Solana Scripts
+
+This architecture describes how the Pipeline runtime supports scripts. 
```

Comment:
This opening line is awkward. It assumes we know what scripts are and that it's somehow desirable to support them.